src/sage/schemes/elliptic_curves/kraus.py

```diff
@@ -108,15 +108,15 @@ def c4c6_model(c4, c6, assume_nonsingular=False):
 
     - ``c4``, ``c6`` -- elements of a number field
 
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
       check for integrality and nosingularity.
 
     OUTPUT:
 
     The elliptic curve with a-invariants [0,0,0,-c4/48,-c6/864], whose
     c-invariants are the given c4, c6.  If the supplied invariants are
```

Comment:
General comment, should we do `c_4` for latex, double backticks for code or leave c4 as c4 in plaintest? This applies to many functions in this file.

Reply:
Probably formatting as LaTeX would be good here, but I'm not going to do it in this PR.
I'll be happy to review a follow-up PR that makes further improvements.

Reply:
Actually I found a moment to take care of some of these markup changes in this file. That's 74c43cd9f82